Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian:stable
+WORKDIR /root/packages
+RUN apt-get update -qq && \
+    apt-get install -y build-essential wget git && \
+    apt-get install -y qemu-system-x86 && \
+    wget https://s3-us-west-2.amazonaws.com/colin.cool/gcc-cross-deb/binutils-x86-64-elf_2.28-5%2B1_amd64.deb && \
+    wget https://s3-us-west-2.amazonaws.com/colin.cool/gcc-cross-deb/gcc-x86-64-elf_6.3.0-18%2B1_amd64.deb && \
```

Comment:
which one? binutils and gcc? something somewhere needs to actually build the kernel....